MonsterTrainModdingAPI/Managers/CustomCharacterManager.cs

```diff
@@ -34,10 +36,13 @@ class CustomCharacterManager
         /// Register a custom character with the manager, allowing it to show up in game.
         /// </summary>
         /// <param name="data">The custom character data to register</param>
-        public static void RegisterCustomCharacter(CharacterData data)
+        /// <param name="info">The Information Used to Load an AssetBundle</param>
+        public static bool RegisterCustomCharacter(CharacterData data, AssetBundleLoadingInfo info = null)
         {
+            if (info != null) CharacterBundleData.Add(data.GetID(), info);
```

Comment:
Please separate these lines. An `if` should not go on the same line as its contents. In general, readability wins against conciseness every single time.